src/Yarp.ReverseProxy.Swagger/ReverseProxyDocumentFilter.cs

```diff
@@ -15,11 +18,14 @@ public sealed class ReverseProxyDocumentFilter : IDocumentFilter
         private readonly IHttpClientFactory _httpClientFactory;
         private ReverseProxyDocumentFilterConfig _config;
         private readonly IReadOnlyDictionary<string, OperationType> _operationTypeMapping;
+        private readonly List<ITransformFactory> _factories;
 
         public ReverseProxyDocumentFilter(
             IHttpClientFactory httpClientFactory,
-            IOptionsMonitor<ReverseProxyDocumentFilterConfig> configOptions)
+            IOptionsMonitor<ReverseProxyDocumentFilterConfig> configOptions,
+            IEnumerable<ITransformFactory> factories)
         {
+            _factories = factories?.ToList() ?? throw new ArgumentNullException(nameof(factories));
```

Comment:
I suggest to delete exception throwing and to add null checking before the line 233. It will keep to work if we don't have any factories.

Reply:
Agree. Done.